unittests/ASM/Disabled_Tests

```diff
@@ -23,3 +23,6 @@ Test_Primary/Primary_E9.asm
 Test_X87/D9_F9.asm
 
 Test_X87/D9_F2.asm
+
+# 3DNow! no longer exists on AMD Zen CPUs
```

Comment:
Now with the `Disabled_Tests_host`, wouldn't it make more sense to move this there?

Reply:
Moved